src/main/java/dev/openfeature/javasdk/EvaluationContext.java

```diff
@@ -1,139 +1,31 @@
 package dev.openfeature.javasdk;
 
-import dev.openfeature.javasdk.internal.Pair;
-import lombok.*;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
 
-import java.time.ZonedDateTime;
-import java.time.format.DateTimeFormatter;
-import java.util.HashMap;
-import java.util.Map;
-
-@ToString @EqualsAndHashCode
+@ToString
+@EqualsAndHashCode
 @SuppressWarnings("PMD.BeanMembersShouldSerialize")
-public class EvaluationContext {
-    @Setter @Getter private String targetingKey;
-    private final Map<String, Pair<FlagValueType, Object>> attributes;
+public class EvaluationContext extends Structure {
+    @Setter
+    @Getter
+    private String targetingKey;
```

Comment:
I'd actually prefer we not split this across 3 lines. `@Setter` isn't enough information to have it's own line, imo.